testing/src/main/java/org/oppia/android/testing/network/MockRetrofitHelper.kt

```diff
@@ -0,0 +1,37 @@
+package org.oppia.android.testing.network
+
+import okhttp3.OkHttpClient
+import org.oppia.android.data.backends.gae.NetworkInterceptor
+import org.oppia.android.data.backends.gae.NetworkSettings
+import retrofit2.Retrofit
+import retrofit2.converter.moshi.MoshiConverterFactory
+import retrofit2.mock.MockRetrofit
+import retrofit2.mock.NetworkBehavior
+import javax.inject.Inject
+
+/**
+ * This helper allows us to create customized [MockRetrofit] instances.
```

Comment:
``Helper used to create specifically configured [MockRetrofit] instances.`` may be clearer.

Also, this can be a one-line KDoc since it's short enough to fit. Ditto for other KDocs.

Reply:
Removed it as now we have `MockRetrofitModule`.

Reply:
Removed it as now we have `MockRetrofitModule`.

Reply:
Removed it as now we have `MockRetrofitModule`.